inc/package.class.php

```diff
@@ -235,7 +235,7 @@ public function prepareInputForUpdate($input) {
                $this->createEntityDirectory(dirname($destination));
                if (rename($uploadedFile, $destination)) {
                   $filename = pathinfo($destination, PATHINFO_FILENAME);
-                  $input['dl_filename'] = basename($destination);
+                  $input['dl_filename'] = basename($uploadedFile);
```

Comment:
I think this is wrong because the downloable file must be the final filename saved, the uploaded file var will give the temporaly uploaded filename set by PHP.